package/proxy/carbonio-proxy.hcl

```diff
@@ -72,6 +72,11 @@ services {
             destination_name   = "carbonio-catalog"
             local_bind_port    = 20012
             local_bind_address = "127.78.0.1"
+          },
+          {
```

Comment:
Is this needed for this change?

Reply:
hi @mbarto not exactly, but I already wanted to express the dependency as we did in other projects